voluncheer/profiles/forms/volunteers.py

```diff
@@ -51,15 +51,14 @@ def save(self, commit=True):
 
 class VolunteerChangeForm(UserChangeForm):
     password = None
-    photo = forms.ImageField(required=False)
+    # photo = forms.ImageField(required=False)
```

Comment:
If the field isn't needed, delete it don't comment it out. Comments like this just bloat and complicate the codebase.